src/machine/machine_rp2040_spi.go

```diff
@@ -231,13 +226,13 @@ func (spi SPI) initSPI(config SPIConfig) (err error) {
 }
 
 //go:inline
-func (spi SPI) setFormat(databits, mode uint8, frameFormat uint32) {
```

Comment:
can we leave the databits argument here? I feel like we are removing valuable information about the API available on the RP2040. We can still remove the DataBits field which I agree is more harmful than good.

Reply:
If the underlying transfer functions don't support anything but 8bits would the ability to set a different value of databits via setFormat be meaning full?

Reply:
It's a purely internal API, I don't see why it should be kept? I think it's only confusing to keep it if it doesn't do anything.

Reply:
I think I agree with @aykevl on this one - to me, a simpler implementation wins.

Reply:
I guess I can agree that from a purely functional standpoint there's no reason to keep databits. Reason I mentioned it was in case someone was to maintain this repo in the future adding 16bit SPI transfer functionality, in which case the `databits` argument would provide intrinsic documentation. I'm fine either way since this is a hypothetical case and the RP2040 documentation is just so gosh darn good by itself.